oneview/resource_storage_system.go

```diff
@@ -185,7 +185,7 @@ func resourceStorageSystem() *schema.Resource {
 func resourceStorageSystemCreate(d *schema.ResourceData, meta interface{}) error {
```

Comment:
These files do not belong to this PR. Can you please check?

Reply:
StorageSystem Struct got renamed, when updated the govendor dependencies, I got these compilation errors so fixed these.

Reply:
you can just checkout the vendor files for storage_system resource as it is not required for this PR. They can be added in the PR for storage system resource.